src/app/app-routing.module.ts

```diff
@@ -1,15 +1,11 @@
-import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { HomePageComponent } from './home-page/home-page.component';
-import { ProfilePageModule } from './profile-page/profile-page.module';
+import { MainComponent } from './profile-page/main/main.component';
 
-const routes: Routes = [
-  {path: '', component: HomePageComponent},
-  {path: 'profile', component: ProfilePageModule}
+const appRoutes: Routes = [
+  { path: 'home', component: HomePageComponent },
+  { path: 'profile', component: MainComponent },
+  { path: '', pathMatch: 'full', redirectTo: 'home' }
 ]
 
-@NgModule({
```

Comment:
Are you sure this is correct? From my understanding, the module needs to have a @NgModule to work. I tried to copy and paste this part of the code on my end and it throws an error.

Reply:
There shouldn't be an @NgModule in the routing, that is already in the app.module.ts and it will cause a duplication error if you don't remove it.